modules/appeals_api/app/models/appeals_api/appellant.rb

```diff
@@ -21,7 +21,7 @@ def last_name
     end
 
     def ssn
-      auth_headers["X-VA#{header_prefix}-SSN"]
+      auth_headers["X-VA#{header_prefix}-SSN"] || auth_headers['X-VA-SSN']
     end
```

Comment:
This accounts for the different name in SC headers.

Reply:
I don't think we should have differing header keys. @cilestin?

Reply:
@cilestin ^^^ ?